### ISSUE TITLE
Verifying error

### ISSUE THREAD

**Issue by MartianCatnip** [derrod/legendary#43]
I'm trying to verify GTA V, but the process freezes at 337/345 (97.7%) and suddenly quits. The computer also lags when hitting that value. I ran using administrator privilege both CMD and legendary.exe

**Comment by derrod** [comment]
Can't reproduce, are you getting any error in the console?

**Comment by MartianCatnip** [comment]
I see something like error 13, it happened very fast before CMD suddenly closing. Sorry I don't really understand Python

**Comment by derrod** [comment]
Error 13 is "Permission denied", not sure why you'd be seeing that. Either way I added a fix in 1ac1875a8654ef5f5059eaf8f2f8d52bc4c11233 that prevents a crash if a file cannot be opened for verifying.

To get the full error run cmd first, then run legendary from within that cmd window, it shouldn't close if just the python process dies.

**Comment by MartianCatnip** [comment]
I re-checked every file manually (MD5 check using CMD). It turns out there's exactly one file that has a wrong MD5, at position 337 out of 345 files (x64q.rpf). I removed that file, and re-run legendary "verify-game". No more error, 0 file corrupted, and 1 missing (as expected). But I still don't understand why something like that causes the error, and maybe others don't have the willingness to check every file, one by one. That's what I found out regarding this problem, hope that helps
